ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -233,6 +235,7 @@ private class PostgresLedgerDao(
         workflowId,
         transaction,
         mappedDisclosure,
+        Map.empty // TODO SC when do we enrich here?
```

Comment:
you should pass the disclosure information in, even if for now we do nothing with it.